server/views/pages/offence/edit-offence.njk

```diff
@@ -48,13 +46,19 @@
                     }
                 }] %}
 
+{% if offence.sentence.countNumber %}
```

Comment:
think this can be pushed into the `{% if offence.sentence %}` rather than outside of it, count is only applicable if an offence has a sentence